src/project/view/exportdialogmodel.cpp

```diff
@@ -306,7 +319,19 @@ void ExportDialogModel::setUnitType(UnitType unitType)
         return;
     }
 
+    bool found = false;
+    for (QVariant availabeUnitType : availableUnitTypes()) {
+        if (availabeUnitType.value<QVariantMap>()["value"].toInt() == static_cast<int>(unitType)) {
+            found = true;
+        }
+    }
+
+    if (!found) {
+        return;
+    }
+
     m_selectedUnitType = unitType;
+    configuration()->setExportUnitType(unitType);
```

Comment:
I'd think the unit type should get the same treatment as the selected export file type, right?

Reply:
Yeah that makes sense